### PR TITLE
combine options topics

### LINKED ISSUE #115: [ REFACTOR ] combine "sepcify advanced runtime options" and "algorithm tuning options"
The `Build/Specify advanced runtime option`s page and the `Optimize/ algorithm tuning options` page should be combined into a single page and any duplication removed. The combined page should be called "Advanced runtime options" and should live in the Run section

### PR DESCRIPTION
**Pull request by beckykd**
closes #115

### CODE REVIEW

File: docs/run/advanced-runtime-options.mdx [hunk 1]
```diff
@@ -1,25 +1,68 @@
...
-    This section focuses on Qiskit Runtime primitive `Options` (imported from ``qiskit_ibm_runtime``). While
-    most of the `primitives` interface is common across implementations, most `Options` are not. Consult the
+    This section focuses on Qiskit Runtime primitive [Options](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.Options) (imported from ``qiskit_ibm_runtime``). While most of the `primitives` interface is common across implementations, most `Options` are not. Consult the
     corresponding API references for information about the ``qiskit.primitives`` and ``qiskit_aer.primitives`` options.
 </Admonition>
 
-## 1. Shots
+## Instantiate the Options class
+
+In the example below, we create an instance of the `Options` class. `optimization_level` is a first level option and can be passed as an input parameter. Options related to the execution environment are passed using the `environment` parameter.
```
Comment (abbycross) [inline docs/run/advanced-runtime-options.mdx:20]:
```suggestion
In the example below, we create an instance of the `Options` class. `optimization_level` is a first-level option and can be passed as an input parameter. Options related to the execution environment are passed using the `environment` parameter.
```

File: docs/run/advanced-runtime-options.mdx [hunk 2]
```diff
@@ -1,25 +1,68 @@
 ---
-title: Algorithm tuning options
-description: Common settings to use when tuning primitive options
+title: Advanced runtime options
```
Comment (Eric-Arellano) [inline docs/run/advanced-runtime-options.mdx:2]: I was going to challenge if this should be called "Advanced"? I see Abby M suggested it.

**Review by abbycross — APPROVED** [review]

File: docs/run/advanced-runtime-options.mdx [hunk 3]
```diff
@@ -1,25 +1,68 @@
...
-One of the advantages of the primitives is that they abstract away the circuit execution setup so that algorithm developers
-can focus on the pure algorithmic components. However, sometimes, to get the most out of an algorithm, you might want
-to tune certain primitive options. This section describes some of the common settings you might need.
+![options](/images/build/options.png)
 
 <Admonition type="info" title="Attention">
-
-    This section focuses on Qiskit Runtime primitive `Options` (imported from ``qiskit_ibm_runtime``). While
-    most of the `primitives` interface is common across implementations, most `Options` are not. Consult the
+    This section focuses on Qiskit Runtime primitive [Options](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.Options) (imported from ``qiskit_ibm_runtime``). While most of the `primitives` interface is common across implementations, most `Options` are not. Consult the
```
Comment (Eric-Arellano) [inline docs/run/advanced-runtime-options.mdx:14]: The double \` is a holdover from RST. It should be a single back tick. I know this was already like this, but it'd be good to fix here. There are a couple throughout the file.

File: docs/run/advanced-runtime-options.mdx [hunk 4]
```diff
@@ -1,25 +1,68 @@
...
+
+In the example below, we create an instance of the `Options` class. `optimization_level` is a first level option and can be passed as an input parameter. Options related to the execution environment are passed using the `environment` parameter.
+
+```python
+from qiskit_ibm_runtime import Options
+
+options = Options(optimization_level=3, environment={"log_level": "INFO"})
+```
+
+The `Options` class supports auto-complete. Once you create an instance of the `Options` class, you can use auto-complete to see what options are available. If you choose one of the categories, you can use auto-complete again to see what options are available under that category.
```
Comment (Eric-Arellano) [inline docs/run/advanced-runtime-options.mdx:28]: I know this is copied over, but it's kind of weird. Opened https://github.com/Qiskit/documentation/issues/181 to investigate in a follow up. No need to do anything now :)

File: docs/run/advanced-runtime-options.mdx [hunk 5]
```diff
@@ -131,3 +174,12 @@ options.resilience_level = 2
 
 estimator = Estimator(session=session, options=options)
 ```
+
+## Next steps
+
+<Admonition type="tip" title="Recommendations">
+    - Find more details about the ``Estimator`` methods in the [Estimator API reference](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.Estimator#estimator).
+    - Find more details about the ``Sampler`` methods in the [Sampler API reference](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.Sampler#sampler).
```
Comment (Eric-Arellano) [inline docs/run/advanced-runtime-options.mdx:182]: Ditto not using two backticks

**Review by Eric-Arellano — APPROVED** [review]
Thanks!

**Pull request merged**